pytorch_lightning/trainer/connectors/debugging_connector.py

```diff
@@ -54,11 +56,18 @@ def on_init_start(
             limit_train_batches = fast_dev_run
             limit_val_batches = fast_dev_run
             limit_test_batches = fast_dev_run
+            self.trainer.max_steps = fast_dev_run
             self.trainer.num_sanity_val_steps = 0
             self.trainer.max_epochs = 1
+            self.trainer.val_check_interval = 1.0
+            self.trainer.check_val_every_n_epoch = 1
+            self.trainer.logger = None
```

Comment:
We may have to replace it with the dummy logger we use for multi-gpu training. If the user makes calls to `trainer.logger.experiment.custom_logging...` their code would brake when setting `fast_dev_run=True`. What do you think?

Reply:
valid point. Updated.